extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -146,7 +146,14 @@ export class MethodsUsageDataProvider
   getChildren(item?: MethodsUsageTreeViewItem): MethodsUsageTreeViewItem[] {
     if (item === undefined) {
       if (this.hideModeledMethods) {
-        return this.sortedTreeItems.filter((api) => !api.method.supported);
+        return this.sortedTreeItems.filter((api) => {
+          const methodCanBeModeled = canMethodBeModeled(
+            api.method,
+            this.modeledMethods[api.method.signature] ?? [],
+            this.modifiedMethodSignatures.has(api.method.signature),
+          );
+          return shouldShowMethod(methodCanBeModeled, this.hideModeledMethods);
```

Comment:
I'm being dumb. `this.hideModeledMethods` is always true here, so this is basically a no-op. The real change here is calling `canMethodBeModeled` and we don't need the change to `shouldShowMethod` at all.